features/element-hiding.json

```diff
@@ -334,7 +338,27 @@
             {
                 "selector": ".sponsored-slot",
                 "type": "hide-empty"
-            }
+            },
+            {
+                "selector": "[id*='M_CONTENTAD']",
+                "type": "closest-empty"
+              },
+              {
+                "selector": "#tfm_admanagerTeaser",
+                "type": "closest-empty"
+              },
+              {
+                "selector": "[id*='M_TRSCT_hor']",
+                "type": "closest-empty"
+              },
+              {
+                "selector": "[id*='M_FLYCPT_hor']",
+                "type": "closest-empty"
+              },
+              {
+                "selector": ".cls_slot_xxoutstreamxx",
+                "type": "hide-empty"
+              }
```

Comment:
These are specific to focus.de, yeah? Let's move them out of the global section so they aren't applied on all pages and instead add them under the `domains` section, eg:
```JSON
{
    "domain": "focus.de",
    "rules": [
        {
            "selector": "[id*='M_CONTENTAD']",
            "type": "closest-empty"
        }
    ]
}
```